setup.py

```diff
@@ -22,16 +22,19 @@
 
 
 with open("trino/__init__.py", "rb") as f:
```

Comment:
Did a number of changes here and below but please note that this is a minimal config. Enabling a strict mode for MyPy or other options triggers dozens of warnings. These should be addressed in separate changes.